README.md

```diff
@@ -1620,12 +1620,12 @@ _For a more comprehensive/advanced/better categorized/... list of Linux audio so
 - [![Open-Source Software][oss icon]](https://github.com/ClementTsang/bottom) [bottom](https://clementtsang.github.io/bottom/nightly/) - Yet another cross-platform graphical process/system monitor.
 - [![Open-Source Software][oss icon]](https://github.com/aristocratos/bpytop) [bpytop](https://github.com/aristocratos/bpytop) - Resource monitor that shows usage and stats for processor, memory, disks, network and processes. Python port and continuation of bashtop.
 - [![Open-Source Software][oss icon]](https://github.com/aristocratos/btop) [btop](https://github.com/aristocratos/btop) - Resource monitor that shows usage and stats for processor, memory, disks, network and processes.
+- [![Open-Source Software][oss icon]](https://github.com/fastfetch-cli/fastfetch) [Fastfetch](https://github.com/fastfetch-cli/fastfetch/) - Fastfetch is a neofetch-like tool for fetching system information and displaying it prettily. It is written mainly in C, with performance and customizability in mind.
```

Comment:
Neofetch was supporting bunch of OS.

I didn't check at fastfetch, but I would expect to list a few of them if the number of OS is smaller.

This could be another reason to keep Neofetch listed also.

Discontinued doesn't mean broken, it means it's not maintained, but it works (for now, and probably a few more years)

Reply:
I can't speak to the difference in this respect, and while I agree that discontinued doesn't mean broken, is there not precedent for removing archived / discontinued projects? For example, #741 which removed projects that no longer work due to API changes (namely, Twitter) but also removed Atom, which may still work, but is archived. A search of PRs for "discontinued" yields more PRs that removed discontinued software, in some cases where the repository was archived and in others where the website was unreachable (#752). 

Having said that, some distributions, such as Arch Linux, [still package Neofetch](https://archlinux.org/packages/extra/any/neofetch/), to my surprise.

I can keep it if there is preference to keep discontinued software in general, perhaps with a caveat that it is discontinued, or if we want to make a special exception here. Neofetch does seem like a bit of an edge-case where it was not discontinued due to a third-party change (e.g. an API change) or slowly phased out (e.g. Atom) so keeping Neofetch with a caveat could be more of an exception rather than adhering to a guideline about how to include software.

Not trying to be argumentative, simply justifying the rationale for removing it based on prior PRs for similar cases, but I can understand if we might want to make an exception for Neofetch.